src/components/common/Directors.vue

```diff
@@ -83,7 +83,7 @@
                     <base-address ref="baseAddressNew"
                       :editing="true"
                       :schema="addressSchema"
-                      @update:address="updateBaseAddress"
+                      @update:address="updateDeliveryAddress"
```

Comment:
This Directors component works fine. This is just a bit of cleanup / commonality with OfficeAddresses component.